apps/cli/src/flags.ts

```diff
@@ -142,5 +142,10 @@ export const buildRunOptions = <T>(y: Argv<T>) => {
       type: "boolean",
       description: "Run codemod in the cloud",
       default: false,
+    })
+    .option("no-interactive", {
+      type: "boolean",
+      description: "Disable interactive mode",
+      default: false,
     });
```

Comment:
we can use already existing "mode" flag for that

Reply:
it is a little different, `mode` is about output
here i'm disabling input, for example when you run `recipe codemod` programmatically and user can't make those choices

Reply:
is there a case where we would want to use both "json" mode and "non-interactive"?